sensor_msgs/Image superpixels
---
sensor_msgs/Image pure_regions